app/ui/src/app/store/integration-support.service.ts

```diff
@@ -73,13 +73,17 @@ export class IntegrationSupportService {
     return this.http.get(url);
   }
 
-  exportIntegration(id: string): Observable<Response> {
-    const url =
-      this.config.getSettings().apiEndpoint +
-      '/integrations/' +
-      id +
-      '/export.zip';
-    return this.http.get(url, { responseType: ResponseContentType.Blob });
+  exportIntegration(...ids: string[]): Observable<Response> {
+    const params = new URLSearchParams();
+    for ( const id of ids ) {
+      params.append('id', id);
+    }
+
+    const url = this.service.one('export.zip').getRestangularUrl();
+    return this.http.get(url, {
+      responseType: ResponseContentType.Blob,
+      params: params
+    });
```

Comment:
The `Http` API was deprecated back in Angular 4.2, and got replaced by the new `HttpClient` class, available from `'@angular/common/http';`. You should use that one instead. The API is almost identical.

All in all, we the UI team are now using the new `ApiHttpService` class, which internally abstracts the API endpoints setup and also provides a lean API to handle multipart requests for uploading files. You can also take advantage of its API here to reduce the overall code footprint for this method.

Reply:
Please submit a PR on to of this one to Convert to newer API.  This PR was not about upgrading usage of older APIs after all.